libs/core/src/components/pds-copytext/pds-copytext.tsx

```diff
@@ -0,0 +1,98 @@
+import { Component, Event, EventEmitter, Host, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'pds-copytext',
+  styleUrl: 'pds-copytext.scss',
+  shadow: true,
+})
+export class PdsCopytext {
+  /**
+   * Determines whether `copytext` should have a border.
+   * @defaultValue true
+   */
```

Comment:
Do we want to reflect this in the DOM?

E.g `{ reflect: true }`

Reply:
I am sure I am totally misunderstanding here, but would using the "bordered" class name be another way to reflect it in the DOM?

Reply:
@ju-Skinner after our convo, I added and double-checked. I totally see what you meant and that's dope! Appreciate the clarification!